website/content/docs/agent/telemetry.mdx

```diff
@@ -542,6 +542,8 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | streams                           | gauge   |
+| `consul.xds.server.idealSessionsMax`                | The ideal maximum number of xDS sessions per server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | sessions                          | gauge   |
+| `consul.xds.server.sessionDrained`                  | Counts the number of xDS sessions that are drained/terminated to rebalance load between servers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | sessions                          | counter |
```

Comment:
I went with the `xds.server` prefix here for consistency with the existing metric rather than the RFC. We also currently refer to xDS sessions as streams here, so I wondered whether we should adopt this in the new metrics for consistency too?